hudi-cli/src/main/java/org/apache/hudi/cli/commands/ArchivedCommitsCommand.java

```diff
@@ -54,7 +55,6 @@
 
   @CliCommand(value = "show archived commit stats", help = "Read commits from archived files and show details")
   public String showArchivedCommits(
-      @CliOption(key = {"archiveFolderPattern"}, help = "Archive Folder", unspecifiedDefaultValue = "") String folder,
```

Comment:
If remove this option, user can not use specified path.

Reply:
> If remove this option, user can not use specified path.

IMO, it is not need here, `archiveFolder` is stored in ` .hoodie`.

Reply:
This config is here to allow for users to be able to provide a full path of the files under the archive folder and read it, lets leave it here - Btw, these commands are being reworked in this PR - https://github.com/apache/incubator-hudi/pull/1274.

Reply:
If leave it there, this pr will has nothing to be commited, just enhancing #1274